Spec/API.md

```diff
@@ -355,7 +355,8 @@ This response only contains metadata (does not include the cards in the deck).
     "created": "<created>"
```

Comment:
Ditto with `PUT /api/v1/deck/<uuid>/flag`, where
```
 {
     user-data-version: <user-data-version>
 }
 ```
should be
```
 {
     "user_data_version": <user-data-version>
 }
 ```

Feel free to address these in a separate PR.